### DISCUSSION TITLE
Islands

### DISCUSSION THREAD

**Discussion by wardpoel** [freshframework/fresh#273] [General]
Hey Fresh people!

What is the reason behind the `islands` name for Island components? Just curious

Thanks! 👨‍💻

**Comment by lucacasonato** [comment]
https://jasonformat.com/islands-architecture/

They are islands of interactivity in a sea of static content.

**Reply by PhoneDroid** [reply]
Please close, docs cover this topic
( https://fresh.deno.dev/docs/concepts/architecture )
